swerve100/src/main/java/org/team100/lib/controller/HolonomicDriveRegulator.java

```diff
@@ -0,0 +1,189 @@
+package org.team100.lib.controller;
```

Comment:
i'd suggest pulling all the repetitive stuff into a separate class, so you'd have one class that encapsulates a single system, controller, etc, and you'd instantiate that class here three times.